modules/collabora_online_group/tests/src/Functional/ViewTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\collabora_online_group\Functional;
+
+use Drupal\Core\Url;
+use Drupal\group\PermissionScopeInterface;
+use Drupal\media\Entity\Media;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\collabora_online\Traits\MediaCreationTrait;
+use Drupal\Tests\collabora_online_group\Traits\GroupRelationTrait;
+use Drupal\Tests\group\Traits\GroupTestTrait;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\user\RoleInterface;
+
+/**
+ * Test the provided view by the module.
+ */
+class ViewTest extends BrowserTestBase {
```

Comment:
Actually `GroupMediaViewsTest`, because this specifically tests the 'group_media' view, which was modified by this module. Also mention this in the doc comment.

Reply:
Indeed, thats better. 

Could be more specific since we don't check all the view, but I think the name becomes too long.